.ci/check_travis_deploy_vars.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -ex
+
+if [[ $TRAVIS_BRANCH == master && $TRAVIS_EVENT_TYPE == push && 
```

Comment:
`"$TRAVIS_BRANCH" == "master" && "$TRAVIS_EVENT_TYPE" == "push"` is the convention, I believe, so that words with spaces or special characters don't break the script.